src/extension.js

```diff
@@ -4,11 +4,35 @@ console.log("Scroll Percentage in Tab Title Extension loaded 👍");
 
 const originalTitle = document.title;
 
-window.addEventListener("scroll", () => {
-  let scrollTop = window.scrollY;
-  let docHeight = document.body.offsetHeight;
-  let winHeight = window.innerHeight;
-  let scrollPercent = scrollTop / (docHeight - winHeight);
-  let scrollPercentRounded = Math.round(scrollPercent * 100);
-  document.title = `(${scrollPercentRounded}%) ${originalTitle}`;
-}, { passive: true });
+window.addEventListener(
+  "scroll",
+  () => {
+    const windowHeight = document.body.offsetHeight;
```

Comment:
This is actually the document height and not the full window height.
Please change the variable name accordingly.
Also, this is a little bit confusing with the `winHeight` variable we have later in the code.